interview-questions.md

```diff
@@ -7,46 +7,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: Animal_id is the foreign key. It would be part of the Sightings model because a sighting belongs to an animal.
 
   Researched answer:
 
 
 
 2. Which routes must always be passed params and why?
 
-  Your answer:
+  Your answer: PUT/PATCH and DELETE routes must always be passed params because these routes always act on a single record. GET can have a param passed when only one record neeeds to be retrieved but it doesn't always need one.
 
   Researched answer:
 
 
 
 3. Based on your knowledge of Rails conventions, write a route for a webpage called "game" that takes in a parameter called "guess" with a controller called "main".
 
-  Your answer:
+  Your answer: GET '/game/guess' = "main#index"
```

Comment:
Totally! The one thing I'd mention is that you forgot the use the colon on guess. E.g. '/game/:guess'